packages/shared/src/appCrypto/modules/pbkdf2.ts

```
@@ -0,0 +1,375 @@
+/* eslint-disable spellcheck/spell-checker */
+import { pbkdf2 as pbkdf2ByNode, pbkdf2Sync as pbkdf2ByNodeSync } from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)